packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInputService.ts

```diff
@@ -33,7 +33,8 @@ export function mapCountryCodeOptions(
   countryCodeOptions: { country: string; code: string }[],
   locale?: string | string[],
 ): Required<SelectProps>['options'] {
-  // TODO: Check whether Intl.DisplayNames is supported and add fallback logic
+  // TODO Intl is only unsupported in Opera 50, iOS Safari 11.0-11.2, Firefox 67, Edge 79, Chrome 63. All of those version are not supported by Circuit UI, so should we even care?
```

Comment:
Do we have to make sure that the code works for unsupported browser versions?

Reply:
What are these support stats based on? 

According to caniuse.com, `Intl.DisplayNames` is [not supported](https://caniuse.com/mdn-javascript_builtins_intl_displaynames) in Chrome <81, Safari <14.1, Firefox <86 and Opera <68.

Reply:
Correct, I've only checked the Intl object instead of the Intl.DisplayNames method, apparently they differ in terms of supported browsers 👍🏻

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames

Reply:
Code updated, the fallback case will use the list of countries passed as keys